### PR TITLE
updates all version to a GM 2 version tag

### PR DESCRIPTION
**Pull request by chrisbsmith**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

* With the advent of maintaining multiple version of our Grey Matter Helm charts, we quickly realized that keeping all published charts in the same `1.X.X` convention, we run the risk of have a fix for Grey Matter 1.0 accidentally overwrite a Grey Matter 2.0 Helm chart.  To prevent this, the Grey Matter 2.0 charts are now versioned as `2.X.X`.  When we release Grey Matter 3.0, we'll update the versions to `3.X.X`.  This will ensure that the charts meant for Grey Matter 2.0 won't be overridden by Grey Matter 1.0 updates.

2. What **changes to custom.yaml** are required?

None

3. Have you documented any additional setup steps or configurations options?

Documented.

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

I've verified that the charts compile but have no manually deployed it yet.

### CODE REVIEW

File: gm-control-api/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.2"
+appVersion: 0.8.1
```
Comment (tilleryd) [inline gm-control-api/Chart.yaml]: good catch

**Comment by tilleryd** [comment]
@chrisbsmith I think we should leave documentation as is in this PR. I'm not sure there's benefit to removing it yet. Also, there's a lot of config left over, if we do decide to remove it, that's enough to warrant another PR

**Comment by tilleryd** [comment]
@chrisbsmith I ran this against staging "fabric-prod" successfully.

**Comment by chrisbsmith** [comment]
@tilleryd  Agree with your comment on documentation. I've added the documentation folder back.

**Pull request merged**